third_party/istio-1.4.3/README.md

```diff
@@ -0,0 +1,13 @@
+The istio\*.yaml files are generated by running
```

Comment:
oh my nope nope nope. We should not add anything about istio into eventing.

Reply:
Istio here is for serving. Let's discuss that in one place: https://github.com/knative/eventing/pull/2388#discussion_r385882618